resources/user-guides/app-extension.md

```diff
@@ -0,0 +1,74 @@
+# iOS App Extension Implementation
+
+You can use supported AEP extensions in your App Extensions to collect usage data. Supported Adobe Extensions are shown [here](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift). This tutorial assumes a basic understanding of how to use the iOS SDK in applications.
+
+## Adding the SDK
+
+Getting the Mobile SDK in your App Extension works the same way as for your application. For this tutorial we will go over Cocoapods as the method of installing the SDK.
+
+1. First, simply add a new target in your podfile for your app extension, then add the Adobe pods to the newly added App Extension target in the podfile.
+
+E.g:
```

Comment:
I don't think we need to say E.g, since it's apparent just from context that the following would be an example.